src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -89,6 +90,12 @@ static void DetectCiphersuiteConfiguration()
     {
         // There's no system_default configuration, so no default CipherString.
         ERR_clear_error();
+
+        if (API_EXISTS(SSL_state))
+        {
+            // OpenSSL 1.0 does not support CipherSuites so there is no way for caller to override default
+            g_config_specified_ciphersuites = 1;
```

Comment:
If the intent is really "OpenSSL 1.0.x should use the OpenSSL default list" (assuming @blowdart said OK), this check could be much earlier and shortcut a lot of work.

Reply:
What is the percentage of platforms stuck on OpenSSL 1.0?

Reply:
I don't have any numbers. AFAIK That covers Ubuntu 16 and some Cernots and RH builds. 

cc: @tmds @omajid

Reply:
RHEL.7 (EOL Aug 2021), CentOS.7 (mainstream support ended, extended ends 2024), Ubuntu 16.04 (EOL April 2021).

Probably anything that hit RTM before ~Mar 2018; which might be just those three.

Reply:
Oh, and the Fedora/RHEL builds will already hit a special case where they have improved the built-in defaults.  So we might only really be talking Ubuntu 16.04.  Debian 9 (Stretch) has already received its last update, and is now only in extended support.

Reply:
> RHEL.7 (EOL Aug 2021), CentOS.7 (mainstream support ended, extended ends 2024), Ubuntu 16.04 (EOL April 2021).

CentOS 7 is based on the RHEL 7 lifecycle. [RHEL 7's EOL would be Aug 2024 (or later)](https://access.redhat.com/support/policy/updates/errata#Life_Cycle_Dates).